include/kernels/contraction_old.cuh

```diff
@@ -0,0 +1,524 @@
+#pragma once
```

Comment:
Can we get rid of this file? I don't see it being included anywhere, but I only looked quickly.

Reply:
addressed dd7c47a5802d829e4cbd89f19149bc1cf86337cd